docs/docs/editor.mdx

```diff
@@ -28,7 +28,7 @@ By design, the `Editor`'s surface area is [very large](/gen/editor/Editor). Almo
 
 This page gives a broad idea of how the `Editor` class is organized and some of the architectural concepts involved. The full reference is available in the [Editor API](/gen/editor/Editor).
 
-## State
+## Store
```

Comment:
I want to use `Store` to refer to the store where possible, so that we don't overload the word 'state' too much.